### PR TITLE
Modified to run on Heroku

### PR DESCRIPTION
**Pull request by jhodges10**
Still runs correctly locally as well as far as I can tell.

### CODE REVIEW

**Comment by jhodges10** [comment]
The latest commit for "added cron entry to keep cache warm" could use a little more work. It's definitely not perfect.

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: NODE_ENV=production node server.js --port $PORT
```
Comment (PastaPastaPasta) [inline Procfile]: AHHHHHH NO NEW LINE!?!?!

Comment (jhodges10) [reply Procfile]: Haha for you, I'll add one @PastaPastaPasta

File: public/loaderio-4d63e500c07b5d7d166e20c374e6534d.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+loaderio-4d63e500c07b5d7d166e20c374e6534d
```
Comment (PastaPastaPasta) [inline public/loaderio-4d63e500c07b5d7d166e20c374e6534d.txt]: AHHHHHHHH

Comment (jhodges10) [reply public/loaderio-4d63e500c07b5d7d166e20c374e6534d.txt]: Dude, I totally added it. I think Github removed it for some reason.

Comment (jhodges10) [reply public/loaderio-4d63e500c07b5d7d166e20c374e6534d.txt]: FIXED IT

**Pull request merged**